var/spack/repos/builtin/packages/py-fenics-basix/package.py

```diff
@@ -34,16 +35,22 @@ class PyFenicsBasix(PythonPackage):
     )
 
     depends_on("fenics-basix@main", type=("build", "run"), when="@main")
+    depends_on("fenics-basix@0.6.0", type=("build", "run"), when="@0.6.0")
     depends_on("fenics-basix@0.5.1", type=("build", "run"), when="@0.5.1")
     depends_on("fenics-basix@0.4.2", type=("build", "run"), when="@0.4.2")
     depends_on("fenics-basix@0.3.0", type=("build", "run"), when="@0.3.0")
     depends_on("fenics-basix@0.2.0", type=("build", "run"), when="@0.2.0")
     depends_on("fenics-basix@0.1.0", type=("build", "run"), when="@0.1.0")
 
+    depends_on("py-setuptools@42:", type=("build", "run"), when="@0.4:")
+    depends_on("py-setuptools@40:", type=("build", "run"))
+    depends_on("py-numpy@1.21:", type=("build", "run"), when="@0.4:")
+    depends_on("py-numpy@1.15:", type=("build", "run"))
     depends_on("cmake@3.19:", type="build")
+    depends_on("python@3.8:", type=("build", "run"), when="@0.4:")
```

Comment:
Shouldn't this be 3.7+? https://github.com/FEniCS/basix/blob/main/python/setup.py#L10